wd_helper.au3

```diff
@@ -3478,19 +3478,21 @@ EndFunc   ;==>__WD_Base64Decode
 
 ; #INTERNAL_USE_ONLY# ===========================================================================================================
 ; Name ..........: __WD_ErrHnd
-; Description ...: Dummy error handler
-; Syntax ........: __WD_ErrHnd()
-; Parameters ....: None
+; Description ...: COM Error handler
+; Syntax ........: __WD_ErrHnd($oError)
+; Parameters ....: $oError              - Error object.
 ; Return values .: None
 ; Author ........: mLipok
 ; Modified ......:
 ; Remarks .......:
-; Related .......:
+; Related .......: __WD_UpdateExtractor, _WD_DownloadAsBinary
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
-Func __WD_ErrHnd()
-
+Func __WD_ErrHnd($oError)
+	Local Const $sFuncName = "__WD_ErrHnd"
+	If $_WD_DEBUG = $_WD_DEBUG_None Then Return
```

Comment:
This line can be eliminated because there is already logic in __WD_Error for this.

Reply:
Please take a look: [REFACTORING](https://github.com/Danp2/au3WebDriver/pull/513/commits/a1f5aee491e44c7c034f4414d626dd2e0cfcfdb0)

Reply:
@mlipok  I was actually referring to this line --
`	If $_WD_DEBUG = $_WD_DEBUG_None Then Return`

It isn't needed because __WD_Error does nothing when $_WD_DEBUG = $_WD_DEBUG_None

Reply:
Please take a look:
[cleanup](https://github.com/Danp2/au3WebDriver/pull/513/commits/2b3437981269bdfa056af2e2c7124016bad8349d)